lib/index.ts

```diff
@@ -43,7 +42,7 @@ function parseArgs() {
   return [target, method, args];
 }
 
-let UUID = 0;
+let UUID = 1;
```

Comment:
Was there an issue with `0` being falsey?

Reply:
yep this there is `if (!timer) { return false; }` in `bb.cancel`